app/api/users/api.py

```diff
@@ -15,14 +22,25 @@
     path="",
     status_code=http_status.HTTP_201_CREATED,
     response_model=CreateUserResponse,
-    dependencies=[Depends(JWTBearer())],
+    dependencies=[],
 )
-async def create_user(data: CreateUserData, user_api: OptScaleUserAPI = Depends()):
+async def create_user(
+    data: CreateUserData,
+    user_api: OptScaleUserAPI = Depends(),
+    jwt_payload: dict = Depends(JWTBearer()),
```

Comment:
Look at my suggestion below. If you agree you could change

```suggestion
    token: str = Depends(JWTBearer(allow_unauthenticated=True)),
```